libglusterfs/src/async.c

```diff
@@ -348,11 +348,11 @@ gf_async_worker_enable(void)
     /* This will wake one of the spare workers. If all workers are busy now,
      * the signal will be queued so that the first one that completes its
      * work will become the leader. */
+    gf_async_worker_create();
```

Comment:
Please, also move the comments to the appropriate lines and add a new comment explaining why the creation of a worker needs to be done before waking other workers.

Reply:
Done